pkg/storage/unified/search/bleve_test.go

```diff
@@ -217,6 +220,23 @@ func TestBleveBackend(t *testing.T) {
 			rsp.Results.Rows[1].Key.Name,
 		})
 
+		// can get sprinkles fields
+		rsp, err = index.Search(ctx, nil, &resource.ResourceSearchRequest{
+			Options: &resource.ListOptions{
+				Key: key,
+			},
+			Limit:  100000,
```

Comment:
maybe add a sort by `DASHBOARD_VIEWS_LAST_1_DAYS` here -- that is the key use case we will need

Reply:
mmm it fails to sort by it 😢 Will fix that

Reply:
wasn't prepending "fields.". I feel like this is going to be an ongoing pain. Could be worth a refactor in the future for how we handle converting field names to bleve field names